.travis.yml

```diff
@@ -233,4 +323,4 @@ before_script:
 
 script: tox -e $TOXENV
 
-after_success: coveralls
+after_success: coveralls
```

Comment:
Please avoid removing newlines at the end of files, that’s better to have them for git.

Reply:
Sorry, editor discrepancies.